### PR TITLE
SSCSCI-2451

### PR DESCRIPTION
**Pull request by hashimalisolirius**
### Jira link
https://tools.hmcts.net/jira/browse/SSCSCI-2451

### Change description
Update venue logic in caseUpdated to handle closed venues. 

### Security Vulnerability Assessment ###

**CVE Suppression:** Are there any CVEs present in the codebase (either newly introduced or pre-existing) that are being intentionally suppressed or ignored by this commit?
  * [ ] Yes
  * [ ] No

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java [hunk 1]
```diff
@@ -448,29 +448,24 @@ private void updateProcessingVenueIfRequired(CaseDetails<SscsCaseData> caseDetai
                     log.info("Processing venue requires updating for case {}: setting venue name to {} from {}",
                             caseDetails.getId(), venue, sscsCaseData.getProcessingVenue());
                     sscsCaseData.setProcessingVenue(venue);
+                    if (isNotEmpty(venue)) {
```
Comment (stuart-hmcts) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java:451]: Do we need a test for when the venue is empty?

**Review by nitinprabhuhmcts — APPROVED** [review]

**Review by nitinprabhuhmcts — APPROVED** [review]

**Review by KatKovacs1 — APPROVED** [review]

**Review by stuart-hmcts — APPROVED** [review]
LGTM
